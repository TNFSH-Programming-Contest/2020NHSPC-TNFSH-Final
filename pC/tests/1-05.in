5 10 4
4 5 747924
5 1 823325
2 1 279321
2 3 574899
3 1 124585
4 3 434129
4 1 714471
3 5 878847
5 2 103688
2 4 298958
