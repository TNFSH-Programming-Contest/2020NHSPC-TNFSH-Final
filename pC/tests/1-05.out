714471 298958 434129 0 747924 
